File: app/src/main/java/com/robotnec/budget/app/adapters/ListHeader.java
package com.robotnec.budget.app.adapters;

import android.support.v7.widget.RecyclerView;
import android.view.ViewGroup;

/**
 * @author zak <zak@swingpulse.com>
 */
public interface ListHeader {
    RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent);

    void onBindViewHolder(RecyclerView.ViewHolder holder);
}

File: app/src/main/java/com/robotnec/budget/app/adapters/support/ListHeader.kt
package com.robotnec.budget.app.adapters.support

import androidx.recyclerview.widget.RecyclerView
import android.view.ViewGroup

/**
 * @author zak zak@swingpulse.com>
 */
interface ListHeader {
    fun onCreateViewHolder(parent: ViewGroup): RecyclerView.ViewHolder

    fun onBindViewHolder(holder: RecyclerView.ViewHolder)
}
